apps/web/src/components/MEVChecker/MEVCheckerProgress.tsx

```diff
@@ -0,0 +1 @@
+export default function MEVCheckerProgress() {}
```

Comment:
if this comes lets mark it as WIP in code or comments to understand for others why this is unfinished and commited 🙏